lib/json_api_client/resource.rb

```diff
@@ -310,21 +354,23 @@ def _build_connection(rebuild = false)
     #
     # @param params [Hash] Attributes, links, and relationships
     def initialize(params = {})
-      params = params.with_indifferent_access
+      params = params.with_indifferent_access # upstream has: params = params.symbolize_keys
```

Comment:
- do we really need access via strings?

- the new upstream versions use `symbolize_keys`

Reply:
Oof, I think this will have some pretty serious implications if we switch to using keys. Likely we'll have to update in several places; which is fine if we're committing to that, just want to point it out

Reply:
@ldrbrandon yes, that's why I left our code in place for now :(

Reply:
https://github.com/rails/rails/blob/cb0c746b7d757d8491f2db1539cf9377b3eb833e/activesupport/lib/active_support/core_ext/hash/indifferent_access.rb#L9